op-service/log/cli.go

```diff
@@ -28,13 +28,13 @@ func CLIFlags(envPrefix string) []cli.Flag {
 		},
 		cli.StringFlag{
 			Name:   FormatFlagName,
-			Usage:  "Format the log output. Supported formats: 'text', 'json'",
+			Usage:  "Format the log output. Supported formats: 'text', 'terminal', 'logfmt', 'json', 'json-pretty',",
 			Value:  "text",
 			EnvVar: opservice.PrefixEnvVar(envPrefix, "LOG_FORMAT"),
 		},
 		cli.BoolFlag{
 			Name:   ColorFlagName,
-			Usage:  "Color the log output",
+			Usage:  "Color the log output if in terminal mode",
 			EnvVar: opservice.PrefixEnvVar(envPrefix, "LOG_COLOR"),
```

Comment:
Previously logs would default to using colour when outputting to the terminal but now it requires specifically specifying `--log.color`.

Reply:
I think that is fine, we would only really want colors if running locally right?

Reply:
I think if we're outputting to a local terminal we should default to using color, otherwise the color functionality will generally not be used.  Definitely agree if it's outputting to logs or anything non-terminal that defaulting to no color makes sense.

Reply:
I believe that it should still use colors by default (even without the flag) when running in a terminal

Reply:
oh, I see. This was a bug in the op-service version, but not in the op-node version

Reply:
Fixed in 3f0d2bd6111acbb32e3a2806b4b2c1dc28cac3df